express/parsers/apps/aiida/parser.py

```diff
@@ -0,0 +1,24 @@
+from express.parsers import BaseParser
+from express.parsers.apps.aiida.formats.zip import AiidaZipParser
+
+from express.parsers.utils import find_fines_by_name_substring
+
+
+class AiidaArchiveParser(BaseParser):
+
+    def __init__(self, *args, **kwargs):
+        super(AiidaArchiveParser, self).__init__(*args, **kwargs)
+        self.path = self.kwargs["workDir"]
+
+    def find_zip_files(self):
+        if self.path is not None:
+            for zip_file_path in find_fines_by_name_substring('.zip', self.path):
+                yield zip_file_path
```

Comment:
This fn might better somewhere in `utils` instead of being part of aiida app, no?

Reply:
We probably want to add more validation here (presence of specific files in the zip-archive), so probably better to extend this directly here.

Reply:
Valid zip- and archive-files are now better detected.